app/about_us/OurCrew/OurCrew.js

```diff
@@ -0,0 +1,66 @@
+import styles from "./OueCrew.module.css";
+
+export default function OurCrew() {
+  // TASK - React 1 week 1
+  // Create the "Our Crew section"
+  // Use the descriptions provided in /app/about_us/README.md
+  // Use the pictures from /public/crew
+  // Some inspiration ideas found in /data/inspiration_about_us
+  return (
+    <>
+      <div className={styles.grid}>
+        <div className={styles.content}>
+          <img
+            className={styles.image}
+            src="./crew/image-anousheh-ansari.png"
+            alt="anushe"
+          />
+          <p>
+            Anousheh Ansari (Persian: انوشه انصاری; née Raissyan;[2] born
+            September 12, 1966) is an Iranian American engineer and co-founder
+            and chairwoman of Prodea Systems
+          </p>
+        </div>
+        <div>
+          <img
+            className={styles.image}
+            src="./crew/image-douglas-hurley.png"
+            alt="anushe"
```

Comment:
This `alt="..."` text is misleading for non-sighted users, so you can either change it or remove it.

- More info on alt tag: https://www.w3.org/WAI/tutorials/images/
- Videos about Accessibility testing, and how people use assistive technologies:
  - Short intro video - https://www.youtube.com/watch?v=HE2R86EZPMA
  - Longer and detailed video from a non-sighted user - https://www.youtube.com/watch?v=OUDV1gqs9GA